docs/core/testing/unit-testing-platform-extensions-vstest-bridge.md

```diff
@@ -32,7 +32,7 @@ The **RunConfiguration** element can include the following elements. None of the
 | **TargetPlatform** | This setting defines the architecture to use to run tests. | `<RuntimeIdentifier>` determines the architecture of the final application that hosts the tests. |
 | **TreatTestAdapterErrorsAsWarnings** | Suppresses test adapter errors to become warnings. | Microsoft.Testing.Platform allows only one type of tests to be run from a single assembly, and failure to load the test framework or other parts of infrastructure will become an un-skippable error, because it signifies that some tests could not be discovered or run. |
 | **TestAdaptersPaths** | One or more paths to the directory where the TestAdapters are located| Microsoft.Testing.Platform does not use the concept of test adapters and does not allow dynamic loading of extensions unless they are part of the build, and are registered in `Program.cs`, either automatically via build targets or manually. |
-| **TestCaseFilter** | A filter to limit tests which will run. | To filter tests use `--filter` command line option. |
+| **TestCaseFilter** | A filter to limit tests which will run. | Before v1.6, use `--filter` command line option. |
```

Comment:
Maybe better to explicitly state that this is supported after 1.6.
The current wording feels like the `--filter` workaround applies to versions earlier than 1.6.